src/client/chowly-api-client.ts

```diff
@@ -29,4 +29,20 @@ export class ChowlyApiClient {
 
         return plainToClass(Sections, response.data);
     }
+
+    async create(order: ICreateOrder): Promise<Order> {
+        const response: AxiosResponse<IOrderResponse> = await this.client.post<IOrderResponse>(`${this.basePrefix}/orders`, order, {
```

Comment:
maybe `IOrderResponse` => `IOrder`?

Reply:
No, server response  I0rder + id type string

Reply:
Can we add `id?: string` to `I0rder`?

Reply:
No

Reply:
Why? is there something else?